tests/test_init.py

```diff
@@ -468,3 +468,48 @@ def test_project_directory_name_variable(runner, temp_dir, project_definition_co
             assert (
                 project_path / "file.txt"
             ).read_text() == f"project directory name: {project_path.name}"
+
+
+def test_snowflake_cli_version_variable(
+    runner, temp_dir, project_definition_copy, monkeypatch
+):
+    project_name = "project_templating"
+    with project_definition_copy(project_name) as template_root:
+        monkeypatch.setattr("snowflake.cli._plugins.init.commands.VERSION", "2.13.7")
+        (template_root / "template.yml").write_text("files_to_render:\n - file.txt")
+        (template_root / "file.txt").write_text("version: <! snowflake_cli_version !>")
+        project = "project"
+        result = runner.invoke(
+            [
+                "init",
+                project,
+                "--template-source",
+                template_root,
+            ]
+        )
+        assert result.exit_code == 0, result.output
+        assert f"Initialized the new project in {project}" in result.output
+        assert (Path(project) / "file.txt").read_text() == f"version: 2.13.7"
+
+
+@pytest.mark.parametrize(
+    "value,expected",
+    [
+        ("", "Users are empty\n"),
+        ("user1,user2,user3", "Users:\n  * user1\n  * user2\n  * user3\n\n"),
+    ],
+)
+def test_jinja_blocks(runner, temp_dir, test_projects_path, value, expected):
+    project_name = "project_templating_jinja_blocks"
+    template_root = test_projects_path / project_name
+    result = runner.invoke(
```

Comment:
Why you didn't use `with project_definition_copy`?

Reply:
In this test I don't need to override `template.yml` - I can simply use the original thing (see other tests, for example `test_default_values`)